ubuntudesign/www.ubuntu.com
# update IoT contact us section

## Done

Updated the contact us form

## QA

Check:
 - `/internet-of-things/contact-us` - [copydoc](https://docs.google.com/document/d/1Se-fD5hhb7EaR75jd_1D-JkADmg6aLjUs7_bScATvwE/edit)
 - `/internet-of-things/thank-you` - [copydoc](https://docs.google.com/document/d/1-WTp-Zf29CA5fiMV0A1rCyaNPF6xAYzZu6BThUXUSlk/edit)
